README.md

```diff
@@ -1 +1,180 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a>https://rubyonrails.org/</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Categories]**
+- **[Transactions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Ruby
+- Install Rails
+- Install Postgresql
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  ```
+  cd Budget-Buddy
+  ```
+  ```
+  git clone git@github.com:deniodev/Budget-Buddy.git
+  ```
+
+
+### Install
+
+Install this project with:
+
+  ```
+  cd Budget-Buddy
+  ```
+  ```
+  bundle install && npm install
+  ```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rubocop --color
+```
+```
+npx stylelint "**/*.{css,scss}"
+```
+```
+rspec 
+```
+
+
+### Deployment
+
+  - The website is deployed [here](https://budget-buddy-yniy.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Denio Nhanale**
+ 
+- GitHub: [@githubhandle](https://github.com/deniodev)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/denionhanale)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Tracking most recent & ancient transactions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/deniodev/Budget-Buddy/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please show support by staring ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+* I am grateful to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) for the priviledge to use his design.
+
+* I would like to thank my Microverse team and my coding partners for their support and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Presentation <a name="presentation"></a>
+
+* [video](https://www.loom.com/share/Budget-Body-Web-App-Presentation-2def64cc00ed494ba3b785f0693beef6?sid=788bbade-d5f7-40db-886c-7c060c8ff33e) presentation.
```

Comment:
[ ] your video presentation should contain a highlight about a piece of the code you find interesting or is proud of making, kindly fix that 🤓.